File: hw2/hw2.py
def type_of_triangle(a, b, c):
    if abs(b-c) >= a or a >= b+c: 
        return "Không tồn tại tam giác"
    if a == b and b == c: 
        return "Tam giác đều"
    elif (a-b)*(b-c)*(c-a) == 0: 
        return "Tam giác cân"
    return "Tam giác thường"

File: hw2/hw2.js
function typeOfTriangle(a, b, c) {
    if (Math.abs(b-c) >= a || a >= b+c) 
        return "Không tồn tại tam giác";
    if (a === b && b === c) 
        return "Tam giác đều";
    else if ((a-b)*(b-c)*(c-a) === 0) 
        return "Tam giác cân";
    return "Tam giác thường";
}

